pkg/init/cmd/service/prepare.go

```diff
@@ -164,6 +165,11 @@ func prepareFilesystem(path string, runtime Runtime) error {
 			return fmt.Errorf("Failed to mount %s: %v", mount.Source, err)
 		}
 	}
+	// relative paths should be relative to container rootfs
+	err := os.Chdir(rootfs)
```

Comment:
IIRC all the container startup happens in the same go process these days, so might this confuse some later thing which expected to be run from `/`?

I think this bit of service launch is not parallel so we are ok from that PoV.

Maybe it would be safer to use `filepath.IsAbs` and `filepoath.Join` though?

Reply:
Yes, probably better.